pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>org.openzal</groupId>
   <artifactId>zal</artifactId>
-  <version>3.19.0</version>
+  <version>3.19.1</version>
```

Comment:
I merged https://github.com/Zextras/OpenZAL/pull/297 that is using version 3.19.1, I think 3.19.2 should be used.

Reply:
thx, updated